src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestCertificate/Cell/BirthdayDatePickerCell.swift

```diff
@@ -0,0 +1,109 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import OpenCombine
+
+class BirthdayDatePickerCell: UITableViewCell, ReuseIdentifierProviding, UITextFieldDelegate {
+
+	// MARK: - Init
+
+	override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+		super.init(style: style, reuseIdentifier: reuseIdentifier)
+		setupView()
+	}
+
+	@available(*, unavailable)
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	// MARK: - Overrides
+
+	// MARK: - Protocol UITextFieldDelegate
+
+	func textFieldShouldClear(_ textField: UITextField) -> Bool {
+		birthdayDate = nil
+		return true
+	}
+
+	// MARK: - Public
+
+	// MARK: - Internal
+
+	@OpenCombine.Published private(set) var birthdayDate: String?
+
+	func configure(
+		placeHolder: String,
+		accessibilityIdentifier: String?
+	) {
+		textField.placeholder = placeHolder
+		self.accessibilityIdentifier = accessibilityIdentifier
+	}
+
+	// MARK: - Private
+
+	private let textField = ENATextField()
+
+	private func setupView() {
+		selectionStyle = .none
+		textField.translatesAutoresizingMaskIntoConstraints = false
+		contentView.addSubview(textField)
+
+		NSLayoutConstraint.activate(
+			[
+				textField.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 15.0),
+				textField.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 15.0),
+				textField.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -15.0),
+				textField.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -15.0),
+				textField.heightAnchor.constraint(equalToConstant: 40.0)
+			]
+		)
+
+		let datePicker = UIDatePicker()
+		datePicker.datePickerMode = .date
+		datePicker.maximumDate = Date()
+
+		if #available(iOS 13.4, *) {
+			datePicker.preferredDatePickerStyle = .wheels
+		}
+		datePicker.addTarget(self, action: #selector(datePickerValueChanged(_:)), for: .valueChanged)
+		textField.inputView = datePicker
+
+		textField.clearButtonMode = .whileEditing
+		textField.delegate = self
+	}
+
+	private lazy var stringDateFormatter: DateFormatter = {
+		let dateFormatter = DateFormatter()
+		dateFormatter.dateFormat = "YYYY-MM-dd"
+		return dateFormatter
+	}()
```

Comment:
I would suggest to use ISO8601Formatter.justDateFormatter. Also prevents us from accidentally using "YYYY" instead of "yyyy" 😉

Reply:
good point, I also move justDate out of the namespace vCard to indicated it gets used in more places now